backend/src/rules/play_card.clj

```diff
@@ -26,13 +26,13 @@
 (defn ^:private apply-play-card
   "Plays a card waiting to be played onto the board"
   [game-state play]
-    (let [player (:player play)
+    (let [player-id (:player play)
```

Comment:
How do we *still* have so many placer where we forgot to put `-id`?

Reply:
I keep changing them whenever I see them.

Reply:
DOING THE LORD'S WORK, SON